docs/developer/developer.adoc

```diff
@@ -317,6 +317,8 @@ behaviour.
 
 * *OSCAP_FULL_VALIDATION=1* - validate all exported documents (slower)
 * *SEXP_VALIDATE_DISABLE=1* - do not validate SEXP expressions (faster)
+* *OSCAP_PCRE_EXEC_RECURSION_LIMIT* - override default recursion limit
+  for match in pcre_exec call in textfilecontent(54) probes.
```

Comment:
I wonder if other probes could be subjects for this problem. But we probably don't have to address them in this PR.

Reply:
Yes, they are, although I think that the textfilecontent is the most dangerous, because there the user provides the regex and the text file can be arbitrary size.

Reply:
Actually, we have many pcre_exec calls across the whole code base, I think it would be better to replace them by some wrapper,  which will always set the limit, and also it can implement printing nice error messages.